public/index.html

```diff
@@ -59,9 +60,10 @@
               <!-- t-message || t-system-message -->
             </div>
             <form action="#" class="c-message-form">
-              <input type="textarea" id="message-input" class="c-input-oval" placeholder="Type your message here...">
```

Comment:
Let's remove the oval class then?

Reply:
Whoops, we still use it in the name modal. :)

Reply:
Let's change it here too.

Reply:
Ummm, change what? You want the input to be styled as oval (as in initial design)?